src/dotnet/commands/dotnet-list/dotnet-list-proj/Program.cs

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Build.Evaluation;
+using Microsoft.DotNet.Cli.CommandLine;
+using Microsoft.DotNet.Cli.Sln.Internal;
+using Microsoft.DotNet.Cli.Utils;
+using Microsoft.DotNet.Tools.Common;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace Microsoft.DotNet.Tools.List.ProjectsInSolution
+{
+    public class ListProjectsInSolutionCommand
```

Comment:
This is so similar to ListProjectToProjectReferencesCommand, with only slight changes between the two. Are we missing some abstraction?